yeat/Snakefile

```diff
@@ -95,18 +95,41 @@ rule downsample:
 
 rule spades:
     output:
-        contigs="analysis/spades/{sample}/contigs.fasta",
-        scaffolds="analysis/spades/{sample}/scaffolds.fasta",
+        contigs="analysis/spades/{sample}_contigs.fasta"
+    input:
+        read1="seq/downsample/{sample}.R1.fq.gz",
+        read2="seq/downsample/{sample}.R2.fq.gz"
+    threads: 8
+    shell:
+        """
+        spades.py -1 {input.read1} -2 {input.read2} -t {threads} -o analysis/spades
+        ln -s scaffolds.fasta {output.contigs}
+        """
+
+
+rule megahit:
+    output:
+        contigs="analysis/megahit/{sample}_contigs.fasta"
     input:
         read1="seq/downsample/{sample}.R1.fq.gz",
         read2="seq/downsample/{sample}.R2.fq.gz",
     threads: 8
```

Comment:
For use on HPC, something more like 64 or 128 would probably be a much more reasonable default here.